File: output/ChatGPT/codenet/C++/Java/s014089689.java
import java.util.Scanner;

public class s014089689 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        String s = scanner.next();
        int cnt = 0;
        for (int i = 0; i < n; i++) {
            cnt++;
            int j = i + 1;
            for (; j < n; j++) {
                if (s.charAt(i) != s.charAt(j)) {
                    break;
                }
            }
            i = j - 1;
        }
        System.out.println(cnt);
    }
}

File: output/ChatGPT/codenet/C++/Python/s014089689.py
n = int(input())
s = input()
cnt = 0
i = 0
while i < n:
    cnt += 1
    j = i + 1
    while j < n:
        if s[i] != s[j]:
            break
        j += 1
    i = j - 1
print(cnt)